android/src/main/java/com/reactlibrary/MyAutoCompleteTextView.java

```diff
@@ -1,25 +1,723 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
 package com.reactlibrary;
 
 import android.content.Context;
+import android.graphics.Rect;
+import android.graphics.Typeface;
+import android.graphics.drawable.Drawable;
+import android.os.Build;
 import android.text.Editable;
 import android.text.InputType;
+import android.text.SpannableStringBuilder;
+import android.text.Spanned;
 import android.text.TextWatcher;
-
-import com.facebook.react.bridge.Arguments;
+import android.text.TextUtils;
+import android.text.method.KeyListener;
+import android.text.method.QwertyKeyListener;
+import android.text.style.AbsoluteSizeSpan;
+import android.text.style.BackgroundColorSpan;
+import android.text.style.ForegroundColorSpan;
+import android.view.Gravity;
+import android.view.KeyEvent;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.inputmethod.EditorInfo;
+import android.view.inputmethod.InputConnection;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.EditText;
+import com.facebook.infer.annotation.Assertions;
 import com.facebook.react.bridge.ReactContext;
-import com.facebook.react.bridge.WritableMap;
-import com.facebook.react.uimanager.events.RCTEventEmitter;
+import com.facebook.react.uimanager.UIManagerModule;
+import com.facebook.react.views.text.CustomStyleSpan;
+import com.facebook.react.views.text.ReactTagSpan;
+import com.facebook.react.views.text.ReactTextUpdate;
+import com.facebook.react.views.text.TextInlineImageSpan;
+import com.facebook.react.views.view.ReactViewBackgroundManager;
+import java.util.ArrayList;
+import javax.annotation.Nullable;
 
 /**
- * Created by rajsuvariya on 7/8/17.
+ * A wrapper around the EditText that lets us better control what happens when an EditText gets
+ * focused or blurred, and when to display the soft keyboard and when not to.
+ *
+ * ReactEditTexts have setFocusableInTouchMode set to false automatically because touches on the
+ * EditText are managed on the JS side. This also removes the nasty side effect that EditTexts
+ * have, which is that focus is always maintained on one of the EditTexts.
+ *
+ * The wrapper stops the EditText from triggering *TextChanged events, in the case where JS
+ * has called this explicitly. This is the default behavior on other platforms as well.
+ * VisibleForTesting from {@link TextInputEventsTestCase}.
  */
```

Comment:
@rajsuvariya line 55 tells us the most interesting piece of info. It says they stop EditText from triggering *TextChanged events. I suspect this is what is causing the autocomplete suggestions to not show.